src/app/login/page.js

```diff
@@ -1,30 +1,33 @@
 "use client";
 import React, { useState } from "react";
 import { Form, Button, Alert } from "react-bootstrap";
+import { login } from "../../../utils/supabase/actions";
 
 const Login = () => {
-  const [inputUsername, setInputUsername] = useState("");
+  const [inputEmail, setInputEmail] = useState("");
   const [inputPassword, setInputPassword] = useState("");
 
-  const [show, setShow] = useState(false);
+  const [error, setError] = useState(false);
```

Comment:
PRAISE: Nice job catching that we don't need that show state with that setShow func, and just replacing it with some state to handle error handling! 

This prevents bloating files with unused or outdated code!